types/sandbox/prerequest.d.ts

```diff
@@ -89,32 +103,59 @@ declare interface Visualizer {
     clear(): void;
 }
 
+declare interface Execution {
+    /**
+     * Halts the execution of current request. No line after this will be executed and
+     * if invoked from a pre-request script, the request will not be sent.
```

Comment:
Need to be more clear for the user. Something like, no scripts associated with the current request will be executed after this and if invoked from a pre-request script, the request will not be sent.

Reply:
No script seems more confusing as even part of current script after this won't be executed is skipRequest invoked.